src/components/ActivityStream.js

```diff
@@ -194,18 +208,28 @@ const ActivityStreamList = ({
           }
 
           const isSeen =
-            DateTime.fromISO(lastSeenAt) > DateTime.fromISO(createdAt);
+            lastSeenAt && createdAt
+              ? DateTime.fromISO(lastSeenAt) > DateTime.fromISO(createdAt)
+              : true;
+
+          const key = hash ? hash : transactionHash;
```

Comment:
This kind of declaration is the same as `const key = hash || transactionHash;`. Here are 3 kind of these definitions